### DISCUSSION TITLE
Status Codes

### DISCUSSION THREAD

**Discussion by frankfrommelt** [Kannix2005/homeassistant-selve#13] [Ideas]
Hi Stefan,

is there a chance to catch the **status codes** that are being returned by COMMEO devices?
Last week I had the situation that one of my covers was stuck.
The error-messages show up on my MultiSend remote, but I did not see anything in Home Assistant.

Best regards
   Frank

**Comment by Kannix2005** [comment]
Well, this is partly implemented but not as trivial. The status codes are read but cannot be displayed in HA atm, because cover entities do not support anything other than positions afaik.
If you have any idea how we could implement this, help me out ;)

**Reply by Kannix2005** [reply]
Thinking about it though, i think i could implement some sensors maybe.

**Comment by frankfrommelt** [comment]
Yes, maybe a "status" sensor could solve this.

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Kannix2005** [comment]
Can you try the new sensor beta release? This should show all sensor values of commeo devices.

**Comment by frankfrommelt** [comment]
Hi Stefan,

OK, I have a number of sensors now.

But they are not changing their values.

All but one of my blinds are set to "Automatic mode", but the value is set to all of them.
Even when I switch between manual and auto mode forth and back.
I have as well testet blocking a blind form closing.
The blind noticed the blockade and moved back.
But all sensors are still set to false.

Take care

    Frank

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Kannix2005** [comment]
Can you try again? Sorry for using you as beta tester ;)

**Comment by frankfrommelt** [comment]
Hi,

well, somethig has changed…

But „Automatic Mode“ stays true whether I switch die blind to “MAN” or “AUTO”.

And blocking the blind does not change the status of any of the sensors.

I am currently using v0.2.18

***@***.***


Viele Grüße

                      Frank Frommelt

Von: Stefan Altheimer ***@***.***>
Gesendet: Mittwoch, 31. Mai 2023 13:52
An: Kannix2005/homeassistant-selve ***@***.***>
Cc: Frank Frommelt ***@***.***>; Author ***@***.***>
Betreff: Re: [Kannix2005/homeassistant-selve] Status Codes (Discussion #13)


Can you try again? Sorry for using you as beta tester ;)

**Comment by frankfrommelt** [comment]
Hi Stefan,

it seems with the current version 0.3.19 the sensor values are changing.

I turned from “manual” to “automatic”.

But it seems this is not happening as soon as I “turn the switch on the device”.

I can see that the RF-USB stick gets a signal (LED flickers) as soon as I turn the switch.

But the status is updated at different point in time.
I guess 8:50 was a re-start of home-assistant.

Take care

    Frank

***@***.***


Viele Grüße

                      Frank Frommelt

Von: Stefan Altheimer ***@***.***>
Gesendet: Mittwoch, 31. Mai 2023 13:52
An: Kannix2005/homeassistant-selve ***@***.***>
Cc: Frank Frommelt ***@***.***>; Author ***@***.***>
Betreff: Re: [Kannix2005/homeassistant-selve] Status Codes (Discussion #13)


Can you try again? Sorry for using you as beta tester ;)

**Comment by Kannix2005** [comment]
Did you see any change in the newest version? (v3.0.1)

**Comment by frankfrommelt** [comment]
Hi Stefan,

sorry for not getting back to you more quickly.

I need to thank you for the good, new version.

Currently it seems to be all working as intended, the sensors update as soon as an event is happening.

Take care

                      Frank

Von: Stefan Altheimer ***@***.***>
Gesendet: Mittwoch, 7. Juni 2023 11:00
An: Kannix2005/homeassistant-selve ***@***.***>
Cc: Frank Frommelt ***@***.***>; Author ***@***.***>
Betreff: Re: [Kannix2005/homeassistant-selve] Status Codes (Discussion #13)


Did you see any change in the newest version? (v3.0.1)
